backend/layers/persistence/persistence.py

```diff
@@ -951,6 +968,32 @@ def replace_dataset_in_collection_version(
 
             return new_dataset_version
 
+    def set_collection_version_datasets_order(
+        self,
+        collection_version_id: CollectionVersionId,
+        dataset_version_ids: List[DatasetVersionId],
+    ) -> None:
+        """
+        Updates the datasets in a collection version to match the given order.
+        """
+        with self._manage_session() as session:
+            collection_version = session.query(CollectionVersionTable).filter_by(id=collection_version_id.id).one()
+
+            # Confirm collection version datasets length matches given dataset version IDs length.
+            if len(collection_version.datasets) != len(dataset_version_ids):
+                raise ValueError(
+                    f"Dataset version IDs length does not match collection version {collection_version_id} datasets length"
```

Comment:
Let's capitalize Dataset Version, Collection Version, Datasets...  same thing in the error message text immediately below 👇